schema/link.schema.json

```diff
@@ -47,13 +50,28 @@
       "type": "boolean"
     }
   },
-  "required": ["href"],
-  "dependencies": {
-    "templated": {
-      "properties": {
-        "href": { "format": "uri-template" },
-        "templated": { "const": true }
+  "required": [
+    "href"
+  ],
+  "if": {
+    "properties": {
+      "templated": {
+        "const": true
+      }
+    }
+  },
+  "then": {
+    "properties": {
+      "href": {
+        "format": "uri-template"
+      }
+    }
+  },
+  "else": {
+    "properties": {
+      "href": {
+        "format": "uri"
```

Comment:
I'm not 100% sure about that one, since we allow relative URIs as well that won't validate against the `uri` value for `format`.

Reply:
I see, thanks.

Reply:
What about JSON Schema's `templated-uri` `string` `format`? Could also be relative URL, right?

Reply:
Indeed, but in my tests relative URLs were rejected for `uri` but not for `uri-template`.

Reply:
I see. Damn!